Dockerfile

```diff
@@ -39,6 +40,4 @@ COPY --from=builder /src/build/libs/nexus-proxy-2.3.0.jar /nexus-proxy.jar
 EXPOSE 8080
 EXPOSE 8443
 
-CMD ["-jar", "/nexus-proxy.jar"]
```

Comment:
adding `JAVA_OPTS` is a great idea, however I think this change would be less impactful to existing users if you just add this to the existing `ENTRYPOINT` options 

I did just a little more research into the topic _(I'm not well versed in Java)_ and found that it looks like the `JAVA_TOOL_OPTIONS` env var is a better fit:
* https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/envvars002.html
* https://stackoverflow.com/a/58355963

Reply:
Thanks, that looks like a less intrusive change. I'll have a go at converting to use the environment variable and update the PR.